playground/src/Playground.tsx

```diff
@@ -23,14 +24,26 @@ const styles = {
 };
 
 type PlaygroundProps = {
+  /**
+  Prevents global styles leak into the playground.
+  */
+  useShadowDOM?: boolean;
+
+  /**
+  The styles to include in the Shadow DOM if useShadowRoot is set to true.
+  */
+  styles?: string | CSSStyleSheet;
+
```

Comment:
originally proposed by @gqio and I kinda blindly followed it without giving good thinking
alternative that I had in mind is to allow define a `<style>` explicitely in code JSX, but it has too many downsides (no adoptedStyleSheets, more verbose, not sure how to make CSSStyleSheet to work even since it's imported from `css` when in `ts/js` context), so I started liking this idea more.